server/src/main/java/org/elasticsearch/common/geo/GeoPoint.java

```diff
@@ -114,6 +118,14 @@ public GeoPoint resetFromCoordinates(String value, final boolean ignoreZValue) {
         return reset(lat, lon);
     }
 
+    private GeoPoint resetFromWKT(String value, boolean ignoreZValue) {
+        try {
+            PointBuilder point = (PointBuilder)GeoWKTParser.parseExpectedType(value, GeoShapeType.POINT, ignoreZValue, false);
```

Comment:
Could you replace it with [WellKnownText](https://github.com/elastic/elasticsearch/blob/8029b479b8b1e1d3cc05bb87d1fdd9ec6325ee1b/libs/geo/src/main/java/org/elasticsearch/geo/utils/WellKnownText.java#L47) parser?